Frontend_Application/src/components/navbar.component.jsx

```diff
@@ -1,31 +1,63 @@
-import React from 'react';
+import React, { useContext } from 'react';
```

Comment:
Is the comma after React supposed to be there?

Reply:
it technically shouldn't be there tbh

Reply:
actually, i think it should - it's telling the file that this is a React file, no?